File: backend/src/main/java/br/ufpr/sept/so2/modules/presenca/application/ports/HostPinPort.java
package br.ufpr.sept.so2.modules.presenca.application.ports;

import java.util.Optional;
import java.util.UUID;

public interface HostPinPort {

    void guardar(UUID eventoId, String pin);

    Optional<String> obter(UUID eventoId);

    void limpar(UUID eventoId);
}

File: backend/src/main/kotlin/br/ufpr/sept/so2/modules/presenca/application/ports/HostPinPort.kt
package br.ufpr.sept.so2.modules.presenca.application.ports

import java.util.Optional
import java.util.UUID

interface HostPinPort {
    fun guardar(eventoId: UUID, pin: String)

    fun obter(eventoId: UUID): Optional<String>

    fun limpar(eventoId: UUID)
}
